apps/protocol-frontend/src/components/ContributionsTable.tsx

```diff
@@ -116,24 +98,27 @@ const ContributionsTable = ({
 
   const data = useMemo<ContributionTableType[]>(() => {
     const tableData = [] as ContributionTableType[];
-    for (const contribution of contributionsData) {
-      tableData.push({
-        name: contribution.name,
-        txHash: contribution.tx_hash,
-        id: contribution.id,
-        details: contribution.details,
-        proof: contribution.proof,
-        date_of_submission: contribution.date_of_submission,
-        engagementDate: contribution.date_of_engagement,
-        attestations: contribution.attestations || null,
-        user: contribution.user,
-        activityTypeId: contribution.activity_type.id,
-        status: contribution.status,
-        action: '',
-        guildName:
-          contribution.guilds.map((guildObj: any) => guildObj.guild.name)[0] ??
-          '---',
-      });
+    for (const page of contributionsData) {
+      for (const contribution of page) {
+        tableData.push({
+          name: contribution.name,
+          txHash: contribution.tx_hash,
+          id: contribution.id,
+          details: contribution.details,
+          proof: contribution.proof,
+          date_of_submission: contribution.date_of_submission,
+          engagementDate: contribution.date_of_engagement,
```

Comment:
Since we don't have the formatting in the ContributionContext now we'll have to do it in the table component:

<img width="1106" alt="image" src="https://user-images.githubusercontent.com/9438776/194466559-a309538f-ec27-4b65-b6a9-859556cdc2b1.png">